packages/wallet/src/ui/Button.tsx

```diff
@@ -25,7 +25,9 @@ import { ButtonOrLink, type ButtonOrLinkProps } from './ButtonOrLink'
 const buttonStyles = cva(['inline-flex items-center justify-center'], {
   variants: {
     intent: {
-      primary: ['text-white bg-gradient-to-r from-green-5 to-green-6'],
+      primary: [
+        'text-white bg-gradient-to-r from-green-5 to-green-6 hover:from-[#7acebe] hover:to-[#2b7576] hover:shadow-md'
```

Comment:
Should we add these colors to tailwind config?🤔

Reply:
I will add them